src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -1011,6 +1019,54 @@ void WindowImplWin32::processEvent(UINT message, WPARAM wParam, LPARAM lParam)
 
             break;
         }
+
+        // File(s) dropped event
+        case WM_DROPFILES:
+        {
+            const HDROP hDrop = (HDROP)wParam;
+
+            // Get the count of files dropped
+            int count = DragQueryFileW(hDrop, 0xFFFFFFFF, NULL, 0);
+            if (count == 0)
+                break;
+
+            // Create a temporary string, which will put all the file names in
+            String item;
+
+            // Get all the files names
+            for (int i = 0; i < count; i++)
+            {
+                std::vector<wchar_t> buffer;
+
+                buffer.resize(DragQueryFileW(hDrop, i, NULL, 0) + 1);
+                DragQueryFileW(hDrop, i, &buffer.front(), static_cast<UINT>(buffer.size()));
+
+                // Insert the file name at the end
+                item.insert(item.getSize(), sf::String(&buffer.front()) + "\n");
+            }
+
+            // Add the temporary item to a vector of strings
+            m_droppedItems.push_back(item);
+
+            // Create the event
+            Event event;
+
+            // Get the position of the dropped item
+            POINT point;
+            DragQueryPoint(hDrop, &point);
+
```

Comment:
```suggestion
            // Get the position of the dropped item
            POINT point;
            DragQueryPoint(hDrop, &point);

            // Create the event
            Event event;
```
Let's try to keep variable declarations as close as possible to where they're used.